contrib/babelfishpg_tsql/src/rolecmds.c

```diff
@@ -1286,8 +1302,14 @@ create_bbf_authid_user_ext(CreateRoleStmt *stmt, bool has_schema, bool has_login
 
 	if (has_login)
 	{
+		char *db_name = get_cur_db_name();
+
 		verify_login_for_bbf_authid_user_ext(login);
 		login_name_str = login->rolename;
+		/* Revoke guest user from login as login now has a mapped user in current database. */
+		grant_revoke_role_to_login(login_name_str, get_guest_role_name(db_name), NULL, false);
+		grant_revoke_role_to_login(login_name_str, get_guest_role_name(db_name), "bbf_role_admin", false); /* revoke even membership granted by bbf_role_admin */
```

Comment:
Current user is set as `bbf_role_admin` when we are here, so not specifying grantor should default to `bbf_role_admin`.

Reply:
+1, Let's verify the number of various grant of guest user (w.r.t. grantor).

Reply:
We need grantor as "bbf_role_admin " for version 16 and above upgrade tests and we can rely on current user for other major version upgrades to version 16.